backend/app/rest/api/rest_private_test.go

```diff
@@ -645,7 +656,7 @@ func TestRest_Email(t *testing.T) {
 	}
 }
 
-func TestRest_EmailNotification(t *testing.T) {
+func TestRest_EmailAndTelegramNotification(t *testing.T) {
```

Comment:
I'm not sure if i like the idea of the mixed tests here. I'd rather prefer setup code extracted or even duplicated and moving tg tests to a separate funcs

Reply:
Addressed in the second commit.